storage/src/main/java/org/apache/kafka/server/log/remote/storage/RemoteLogManager.java

```diff
@@ -2054,6 +2054,8 @@ public void close() {
                 Utils.closeQuietly(indexCache, "RemoteIndexCache");
                 Utils.closeQuietly(remoteLogMetadataManagerPlugin, "remoteLogMetadataManagerPlugin");
                 Utils.closeQuietly(remoteStorageManagerPlugin, "remoteStorageManagerPlugin");
+                Utils.closeQuietly(fetchQuotaMetrics, "fetchQuotaMetrics");
+                Utils.closeQuietly(copyQuotaMetrics, "copyQuotaMetrics");
```

Comment:
minor suggestions: 

shall we move these 2 lines to RemoteLogManager#removeMetrics() method? And, call RemoteLogManager#removeMetrics()  at the end after closing `remoteStorageManagerPlugin`. 

The try..finally block in L2045-2049 can also be removed.